collab-service/src/message_handler/room/connect_message_builder.ts

```diff
@@ -15,6 +15,12 @@ function encodeContentOpcode(content: string, opcode: number): Uint8Array {
   return encoding.toUint8Array(encoder);
 }
 
+function encodeOpcodeOnly(opcode: number): Uint8Array {
+  const encoder = encoding.createEncoder();
+  encoding.writeUint8(encoder, opcode);
+  return encoding.toUint8Array(encoder);
+}
+
```

Comment:
Could be simplified.
```suggestion
function encodeOpcodeOnly(opcode: number): Uint8Array {
  return new Uint8Array([opcode]);
}

```